docs/Makefile

```diff
@@ -1,20 +1,180 @@
-# Minimal makefile for Sphinx documentation
+# Makefile for Sphinx documentation
```

Comment:
Did this come from somewhere as a template? Can you add a reference if so.

Reply:
Indeed! From the python project template:

https://github.com/ethereum/ethereum-python-project-template/blob/main/docs/Makefile